src/fava/core/__init__.py

```diff
@@ -222,7 +223,7 @@ class FavaLedger:
     #: Dict of list of all (unfiltered) entries by type.
     all_entries_by_type: EntriesByType
 
-    def __init__(self, path: str) -> None:
+    def __init__(self, path: str, poll_watcher: bool = False) -> None:
```

Comment:
If you make this a keyword-only argument, the linter won't complain (since it forces the caller to not accidentally pass a boolean arg without telling which one it is)